package.json

```diff
@@ -29,6 +29,7 @@
     "clean": "npm run clean:build",
     "clean:docs": "rimraf docs/api/*",
     "clean:build": "rimraf build",
+    "flow": "flow; test $? -eq 0 -o $? -eq 2",
```

Comment:
I am using the [recommended setup](https://flowtype.org/docs/getting-started.html#_).